CMakeLists.txt

```
@@ -97,7 +97,8 @@ add_feature_info(MPICommunication PRECICE_MPICommunication
 add_feature_info(PETScMapping PRECICE_PETScMapping
   "Enables the PETSc-powered radial basic function mappings.
 
-   The radial basis function mappings require MPI and PETSc to work in parallel.
+   This enables the PETSc variant of radial basis function mappings which is highly recommended 
+   for large cases running in parallel.
```

Maybe this could be more specific? Something in the lines of:

```suggestion
   This enables the PETSc-based variant of radial basis function mappings which can run in parallel,
   also across different compute nodes. This is highly recommended for large cases running in parallel.
```

Otherwise, it is not really clear "why?".